tests/unit/test_operations/test_operation.py

```diff
@@ -0,0 +1,32 @@
+from relevanceai.dataset import Dataset
+
+from sklearn.cluster import KMeans
+
+
+class TestOperation:
+    def test_subcluster(self, test_dataset: Dataset):
+        model = KMeans(n_clusters=4)
+
+        vector_field = "sample_1_vector_"
+        alias = "cluster_test_1"
+        test_dataset.cluster(
+            model=model,
+            vector_fields=[vector_field],
+            alias=alias,
+            include_cluster_report=False,
+        )
+        assert f"_cluster_.{vector_field}.{alias}" in test_dataset.schema
+
+        parent_field = f"_cluster_.{vector_field}.{alias}"
+        vector_field = "sample_2_vector_"
+        alias = "subcluster_test_1"
+        test_dataset.subcluster(
+            model=model,
+            alias=alias,
+            vector_fields=[vector_field],
+            parent_field=parent_field,
+            cluster_ids=[f"_cluster.{vector_field}.{alias}"],
+            filters=["sample_1_vector"],
+            min_parent_cluster_size=2,
+        )
+        assert parent_field in test_dataset.schema
```

Comment:
We need to make sure the subcluster output is in the schema. 
The subcluster output should look something like: 

```
f"_cluster_.{vector_field}.{alias}"
```

Reply:
https://github.com/RelevanceAI/RelevanceAI/pull/780/commits/13e6ad074ecb6859d5a37e853618830fe7e3210d